### PR TITLE
change deprecated io/ioutil to io and os

### PR DESCRIPTION
**Pull request by HeavyRain266**
This pull request removes use of currently deprecated `io/ioutil`, everything from this wrapper has been moved to `io` and `os`.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,8 @@ module github.com/minio/simdjson-go
 go 1.13
```
Comment (harshavardhana) [inline go.mod:3]: If you are doing that @HeavyRain266 you need to make sure the minimum go is 1.16

**Review by harshavardhana — CHANGES_REQUESTED** [review]
We wanted this library to support older Go as well

File: go.mod (hunk 1 above)
Comment (HeavyRain266) [reply go.mod:3]: sorry, I forgot to look at this place.

**Comment by HeavyRain266** [comment]
Go 1.17 is comming out next week and 1.13 is quite old now but I can just close if you still want to support old go

**Comment by klauspost** [comment]
@HeavyRain266 Let's wait until these versions are at least EOL.

**Pull request closed without merging**